src/SynapseAdminApis.ts

```diff
@@ -52,6 +52,11 @@ export interface SynapseUserProperties extends SynapseUser {
      * The password for the user. Leave undefined to leave unchanged.
      */
     password?: string;
+
+    /**
+     * If set to false, devices aren't logged out even when a password is provided.
```

Comment:
```suggestion
     * If set to `false`, devices aren't logged out even when `password` is provided.
```

Reply:
Typically I'd agree with monospacing `false`, but there are plenty of other docstrings in this file that don't do it, so I'll keep the formatting as-is for consistency.

Though what I will do is:
- Reword this to "If false...", which is done elsewhere in this file (consistency strikes again!)
- Put `password` in a `@link` tag